docs/content/documentation/systems/Materials/heat_conduction/SemiconductorLinearConductivity.md

```diff
@@ -0,0 +1,25 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+
+# SemiconductorLinearConductivity
+!syntax description /Materials/SemiconductorLinearConductivity
+
+For Linear relationship in Steinhart Hart equation, coefficient C = 0 \\
+Ref: https://en.wikipedia.org/wiki/Electrical_resistivity_and_conductivity \\
+
+$$\frac{1}{T} = A + B \ln(\rho)$$
```

Comment:
We have to use latex style \begin{equation}, \end{equation} now for out-of-line equations.